src/components/Testimonial/Testimonial.types.ts

```diff
@@ -0,0 +1,9 @@
+export type TestimonialProps = {
+  image: string;
+  label: string;
+  labelBold: string;
+  description: string;
+  isMobile: boolean;
+};
+
+export type TestimonialVariantProps = Omit<TestimonialProps, 'isMobile'>;
```

Comment:
I remember when you posted the cheatsheet with utility types, love to see them in action 🏎️